pytest_tagging/plugin.py

```diff
@@ -48,6 +58,25 @@ class TaggerRunner:
     def __init__(self, counter_class: type[Counter] | type[TagCounterThreadSafe]) -> None:
         self.counter = counter_class()
 
+    def get_available_tags(self, items, print_info=True) -> list:
+        """
+        Returns all available tags
+        :param items: Items from pytest_collection_modifyitems
+        :param print_info: If True, prints all available tags
+        """
+        available_tags = []
+        for item in items:
+
+            test_tags = set(get_tags_from_item(item))
+            for tag in test_tags:
+                if tag not in available_tags:
+                    available_tags.append(tag)
+        if print_info:
+            print("Available tags:")
+            print("\n".join(available_tags))
```

Comment:
Instead of prints (which may be eaten up by pytest if not using `-s`, can we use the terminalreporter like in here: https://github.com/scastlara/pytest-tagging/blob/main/pytest_tagging/plugin.py#L83?

Reply:
It should be possible. 
I made the code that should work for it but sadly It showed some issues that the other tests didn't catch.
If you don't use `--tags` when running it will still print the tags available which does make sense!
Back to the drawing board..

Reply:
If it's too hard we can always leave it like this for now and improve on it later if we can. The feature adds value as it is now.